rel/files/app.config

```diff
@@ -13,7 +13,8 @@
            {format_packet_filter, {fun mongoose_log_filter:format_packet_filter/2, no_state}},
            {format_acc_filter, {fun mongoose_log_filter:format_acc_filter/2, no_state}},
            {format_c2s_state_filter, {fun mongoose_log_filter:format_c2s_state_filter/2, no_state}},
-           {format_stacktrace_filter, {fun mongoose_log_filter:format_stacktrace_filter/2, no_state}}
+           {format_stacktrace_filter, {fun mongoose_log_filter:format_stacktrace_filter/2, no_state}},
+           {format_toml_filter, {fun mongoose_log_filter:format_toml_filter/2, no_state}}
```

Comment:
Good.
An alternative could be, because it could be not the first field with that issue:

```
{format_term_filter, {fun mongoose_log_filter:format_term_filter/2, [toml_value]}}
```

Reply:
Why not, I will do that, I am only concerned about `app.config` becoming the new `mongooseim.cfg` with all the lower level options. This might be the way to go though as long as the settings here are not expected to be changed by non-Erlang users like devops.

Reply:
Yep, I was thinking about asking **MongooseIM** to create handlers after the start and keep app.config minimal.
And after reading logger docs carefully, it looks like one of use cases OTP team had in mind.


But if we want to configure logging, a very good place would be to put options into mongooseim.toml, but it would limit logging usage until configuration parser has finished its work.

Alternative I see is to put `[logging]` block into toml and parse it first at start to get loglevel and filenames for logs, init logging subsystem dynamically using https://erlang.org/doc/man/logger.html#add_handler-3 and continue parsing all other options (which are more likely to contain user-made mistakes, that would need reporting).

While I like that approach, it could be hard to do with the monolithic parser as we have now, so it would be interesting to discuss :)